app/src/main/java/com/github/k409/fitflow/ui/screen/friends/FriendRequestScreen.kt

```diff
@@ -133,6 +219,35 @@ fun FriendRequestContent(
                 modifier = Modifier.padding(start = 8.dp),
             )
         }
+
+        if (searchTextByName.length >= 3 && searchResultsByName.isNotEmpty()) {
+            Column(
+                modifier = Modifier
+                    .padding(vertical = 8.dp)
+                    .padding(start = 18.dp, end = 12.dp),
+            ) {
+                searchResultsByName.forEach { user ->
```

Comment:
There might be an issue where if there are hundreds of search results, then the whole screen will get filled up with search results and user would need to scroll down a lot to add a friend. Also, search results do not go away if user selects an email or name. 
You could try to resolve this by modifying your existing code or rework searching input fields using Material 3 SearchBar composable, which solves most of these problems.
https://m3.material.io/components/search/overview

Reply:
@GillWell you can take reference of [this code](https://github.com/utkarsh006/Drinks/blob/main/app/src/main/java/com/example/hoichi/presentation/drink_list_screen/components/SearchComponent.kt) for searchbar